airbyte-webapp/src/services/connector/ConnectorService.ts

```diff
@@ -1,11 +1,15 @@
 import { useConfig } from "config";
-import { webBackendCheckUpdates } from "core/request/AirbyteClient";
+import { webBackendCheckUpdates, WebBackendCheckUpdatesRead } from "core/request/AirbyteClient";
 import { AirbyteRequestService } from "core/request/AirbyteRequestService";
 import { useDefaultRequestMiddlewares } from "services/useDefaultRequestMiddlewares";
 import { useInitService } from "services/useInitService";
+import { isCloudApp } from "utils/app";
```

Comment:
This is great as a hotfix—small, simple, direct, effective—but I do think that a good follow-up would be editing the conditional to check if `FeatureItem.AllowUpdateConnectors` is enabled instead of checking `isCloud()`. It looks like we don't really have an API for that besides `useFeature`, which is a bummer: right now this module doesn't introduce any react-specific context like hooks impose.

Reply:
Doing conditionals with hooks always gets awkward, but yeah.
We're going to revert the backend change Monday so maybe @josephkmh can make that change if he gets to it before I'm online!

Reply:
@ambirdsall agreed - I have an open issue [here](https://github.com/airbytehq/airbyte-cloud/issues/3972) already proposing that approach. I'll keep that issue open as a refactor of this hotfix.